proposals/incubation/kubeflow.md

```diff
@@ -0,0 +1,178 @@
+
+The current snapshot of the maintainers as defined above (company affiliation based on user public GitHub profile). 
+
+* Kubeflow
+    * [pdmack](https://github.com/pdmack) - Pete MacKinnon (NVIDIA)
+    * [james-jwu](https://github.com/james-jwu) - James Wu (Google)
+    * [zijianjoy](https://github.com/zijianjoy) - James Liu (Google)
+* Kubeflow Notebooks & Common Kubeflow Components
+    * [elikatsis](https://github.com/elikatsis) - Ilias Katsakioris (Arrikto)
+    * [kimwnasptd](https://github.com/elikatsis) - Kimonas Sotirchos (Arrikto)
+    * [StefanoFioravanzo](https://github.com/StefanoFioravanzo) - Stefano Fioravanzo (Arrikto)
+    * [thesuperzapper](https://github.com/thesuperzapper) - Mathew Wicks
+    * [yanniszark](https://github.com/yanniszark) - Yannis Zarkadas (Arrikto)
+* Kubeflow Manifests
+    * [elikatsis](https://github.com/elikatsis) - Ilias Katsakioris (Arrikto)
+    * [kimwnasptd](https://github.com/elikatsis) - Kimonas Sotirchos (Arrikto)
+    * [PatrickXYS](https://github.com/PatrickXYS) - Yao Xiao (Google)
+    * [StefanoFioravanzo](https://github.com/StefanoFioravanzo) - Stefano Fioravanzo (Arrikto)
+    * [yanniszark](https://github.com/yanniszark) - Yannis Zarkadas (Arrikto)
+* Pipelines
+    * [Animeshsingh](https://github.com/animeshsingh) - Animesh Singh (IBM)
+    * [chensun](https://github.com/chensun) - Chen Sun (Google)
+    * [connor-mccarthy](https://github.com/connor-mccarthy) - Connor McCarthy (Google)
+    * [gkcalat](https://github.com/gkcalat) - Ablai Akhazhanov (Google)
+    * [IronPan](https://github.com/IronPan) - Yang Pan (Google)
+    * [james-jwu](https://github.com/james-jwu) - James Wu (Google)
+    * [jlyaoyuli](https://github.com/jlyaoyuli) - Joe Li (Google)
+    * [Linchin](https://github.com/Linchin) - Lingqing Gan (Google)
+    * [zijianjoy](https://github.com/zijianjoy) - James Liu (Google)
+* Training-operator
+    * [Gaocegege](https://github.com/Gaocegege) - Ce Gao (TensorChord)
+    * [Jeffwan](https://github.com/Jeffwan) - Jiaxin Shan (Bytedance)
+    * [Johnugeorge](https://github.com/Johnugeorge) - Johnu George (Nutanix)
+    * [Terrytangyuan](https://github.com/Terrytangyuan) - Yuan Tang (Akuity)
+    * [zw0610](https://github.com/zw0610) - Wang Zhang (Tencent)
+* Katib
+    * [Andreyvelich](https://github.com/Andreyvelich) - Andrey Velichkevich (Apple)
+    * [Gaocegege](https://github.com/Gaocegege) - Ce Gao (TensorChord)
+    * [Johnugeorge](https://github.com/Johnugeorge) - Johnu George (Nutanix)
+    * [tenzen-y](https://github.com/tenzen-y) - Yuki Iwai (CyberAgent)
```

Comment:
Please add github.com/kubeflow/mpi-operator as well

Reply:
+1

Reply:
I have suggested changes. @rochaporto @mrbobbytables The reason for this change is that we want to make it clear that MPI Operator is also donated to CNCF. It's currently a standalone subproject since it has wider range of use cases besides model training.

Reply:
Thanks @terrytangyuan , but i don't think this is required. I see the mpi-operator under the kubeflow org and copyright set to 'The Kubeflow Authors'. If it's a subproject it's clear it's included.

Reply:
I've taken your suggestions just in case.